www-cronrobot/Gemfile

```diff
@@ -64,7 +64,7 @@ group :development do
 end
 
 group :production do
```

Comment:
add a class when the menu item is active. just make the class and put anyone as active, i'll do the rest in rails

![image](https://user-images.githubusercontent.com/8007722/132593291-bbbb9ca1-f1be-4fe1-800d-be621b0ce39d.png)